log/loki_test.go

```diff
@@ -101,6 +108,77 @@ func TestSyslogFromConfigLine(t *testing.T) {
 	}
 }
 
+func TestParseArray(t *testing.T) {
+	cases := [...]struct {
+		key, value string
+		i          int
+		args       []string
+		result     []string
+		resultI    int
+		err        bool
+	}{
+		{
+			key:     "test",
+			value:   "[some",
+			i:       2,
+			args:    []string{"else=asa", "e=s", "test=[some", "else]"},
```

Comment:
These tokens don't look right to me, since you're treating the comma separating each value as the same separator for array values, which goes back to the convoluted parsing comment.

So I'd prefer to see `[]string{"else=asa", "e=s", "test=[some,else]"}` here.

Reply:
see https://github.com/loadimpact/k6/pull/1639/files#r495747384